### ISSUE TITLE
Converting POSIX time zone string to TimeZoneInfo 

### ISSUE THREAD

**Issue by EricStG** [mj1856/TimeZoneConverter#69]
Is it possible to use `PosixTimeZone` to convert a POSIX time zone string to a `TimeZoneInfo` so it can be used with time conversion methods?

**Comment by mj1856** [comment]
No, sorry that's not currently possible.  I'll keep it as a feature request though.  The implementation would have to parse details out of a POSIX string and use `TimeZoneInfo.CreateCustomTimeZone`.
